edx_name_affirmation/tests/test_tasks.py

```diff
@@ -0,0 +1,59 @@
+"""
+Tests for Name Affirmation tasks
+"""
+
+from celery.exceptions import Retry
+from mock import patch
+
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from edx_name_affirmation.models import VerifiedName
+from edx_name_affirmation.statuses import VerifiedNameStatus
+from edx_name_affirmation.tasks import idv_update_verified_name, proctoring_update_verified_name
+
+User = get_user_model()
+
+
+class TaskTests(TestCase):
+    """
+    Tests for tasks.py
+    """
+    def setUp(self):  # pylint: disable=super-method-not-called
+        self.user = User(username='tester', email='tester@test.com')
+        self.user.save()
+        self.verified_name_obj = VerifiedName(
+          user=self.user, verified_name='Jonathan Doe', profile_name='Jon Doe',
+        )
+        self.verified_name_obj.save()
+        self.idv_attempt_id = 1111111
+        self.proctoring_attempt_id = 2222222
+
+    @patch('edx_name_affirmation.tasks.idv_update_verified_name.retry')
+    def test_idv_retry(self, mock_retry):
+        mock_retry.side_effect = Retry()
```

Comment:
Is having a `side_effect` necessary if we aren't expecting the call to retry to produce anything?

Reply:
Looks like no, I can remove that